wcivf/apps/elections/models.py

```diff
@@ -493,6 +493,26 @@ def postal_vote_application_deadline(self):
         except AttributeError:
             return None
 
+    @property
```

Comment:
This code is bobbins, but I wrote it anyway.

We should totally fix this
https://app.asana.com/0/1207876275349083/1208707350429290

but we decided to ignore it for a bit so I'm just following the same pattern for this field, even though I don't like it.